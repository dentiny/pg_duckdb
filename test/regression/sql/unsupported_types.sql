CREATE TEMP TABLE numeric_tbl (a NUMERIC) USING duckdb;
