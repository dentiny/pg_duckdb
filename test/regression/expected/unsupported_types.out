CREATE TEMP TABLE numeric_tbl (a NUMERIC) USING duckdb;
ERROR:  Unsupported type when creating column: type precision 40 with scale 5 is not supported by duckdb, which only allows maximum precision 38
